src/index.js

```diff
@@ -77,70 +74,85 @@ function handleBrowsersBadge(req, res, browsersList) {
  * - /sauce/:user - Get jobs for any build (regardless of CI service).
  * - /travis/:user/:repo/sauce - Get jobs for a Travis build.
  */
-// eslint-disable-next-line max-params
-function handleSauceBadge(req, res, client, source, sauceJobs) {
-  let browsers;
+const getSauceBadgeBody = ({ req, sauce, source, sauceJobs }) => {
```

Comment:
Feel like this function could use some `async/await` magic, but it's not urgent!

Reply:
Yeah, totally. I got nervous refactoring it more since most of the logic is constructing that `getBrowsers` promise (definitely a promise in this context 😉 ).